examples/portfolio/src/components/Nav.astro

```diff
@@ -1,99 +1,324 @@
-		text-transform: uppercase;
-		padding-top: 0.75em;
-		padding-bottom: 0.75em;
 	}
 
-	.link:hover,
-	.link:focus {
-		color: var(--t-active);
+	.link.active {
+		color: var(--gray-0);
 	}
 
-	.monogram {
+	.menu-footer {
+		--icon-size: var(--text-xl);
+		--icon-padding: 0.5rem;
+
 		display: flex;
-		align-items: center;
-		justify-content: center;
-		width: 2em;
-		height: 2em;
-		margin-right: 0.5rem;
-		color: var(--c-black);
-		font-weight: 900;
-		letter-spacing: -0.125rem;
-		border: 3px solid currentColor;
-		border-radius: 50%;
+		justify-content: space-between;
+		gap: 0.75rem;
+		padding: 1.5rem 2rem 1.5rem 1.5rem;
+		background-color: var(--gray-999);
+		border-radius: 0 0 0.75rem 0.75rem;
+		box-shadow: var(--shadow-lg);
 	}
 
 	.socials {
 		display: flex;
-		gap: 0.75em;
-		margin-left: auto;
+		flex-wrap: wrap;
+		gap: 0.625rem;
+		font-size: var(--icon-size);
 	}
 
 	.social {
-		display: block;
+		display: flex;
+		padding: var(--icon-padding);
+		text-decoration: none;
+		color: var(--accent-dark);
+		transition: color var(--theme-transition);
+	}
+
+	.social:hover,
+	.social:focus {
+		color: var(--accent-text-over);
+	}
+
+	.theme-toggle {
+		display: flex;
+		align-items: center;
+		height: calc(var(--icon-size) + 2 * var(--icon-padding));
 	}
 
-	.socialIcon {
-		display: block;
-		width: 1.25rem;
-		height: 1.25rem;
-		fill: var(--t-subdue);
-		transition: fill linear 150ms;
+	@media (min-width: 50em) {
+		nav {
+			display: grid;
+			grid-template-columns: 1fr auto 1fr;
+			align-items: center;
+			padding: 2.5rem 5rem;
+			gap: 1rem;
+		}
+
+		.menu-header {
+			padding: 0;
+		}
+
+		.site-title {
+			font-size: var(--text-lg);
+		}
+
+		#menu-content {
+			display: contents;
+		}
+
+		.nav-items {
+			position: relative;
+			flex-direction: row;
+			font-size: var(--text-sm);
+			border-radius: 999rem;
+			border: 0;
+			padding: 0.5rem 0.5625rem;
+			background: radial-gradient(var(--gray-900), var(--gray-800) 150%);
+			box-shadow: var(--shadow-md);
+		}
+
+		.nav-items::before {
+			position: absolute;
+			inset: -1px;
+			content: '';
+			background: var(--gradient-stroke);
+			border-radius: 999rem;
+			z-index: -1;
+		}
+
+		.link {
+			padding: 0.5rem 1rem;
+			border-radius: 999rem;
+			transition: color var(--theme-transition), background-color var(--theme-transition);
+		}
+
+		.link:hover,
+		.link:focus {
+			color: var(--gray-100);
+			background-color: var(--accent-subtle-overlay);
+		}
+
+		.link.active {
+			color: var(--accent-text-over);
+			background-color: var(--accent-regular);
+		}
+
+		.menu-footer {
+			--icon-padding: 0.375rem;
+
+			justify-self: flex-end;
+			align-items: center;
+			padding: 0;
+			background-color: transparent;
+			box-shadow: none;
+		}
+
+		.socials {
+			display: none;
+		}
 	}
 
-	.socialIcon:hover {
-		fill: var(--t-active);
+	@media (min-width: 60em) {
+		.socials {
+			display: flex;
+			justify-content: flex-end;
+			gap: 0;
+		}
 	}
 </style>
```

Comment:
More high-contrast specific styles to fix the current page styles.
| Before | After |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61414485/210802057-9c93d3bb-9f50-409a-a06c-5fabff7e819f.png) | ![image](https://user-images.githubusercontent.com/61414485/210802077-45492ca4-f73b-45df-a943-0292fc942a9f.png) |

Note that I choose to change the text color instead of the `background-color` because it was causing a weird background over the text and also it wasn't looking good on the hamburger menu:

![image](https://user-images.githubusercontent.com/61414485/210802658-929aebdc-e399-4f68-8a01-18d3ffea85ca.png)

![image](https://user-images.githubusercontent.com/61414485/210802706-9a6815b2-72fc-4857-ab12-6bd2fc1cd902.png)

```suggestion
	@media (forced-colors: active) {
		.link.active {
			color: SelectedItem;
		}
	}
</style>
```